section_1/responseGuiApplication/webmanager.py

```diff
@@ -181,19 +254,24 @@ def pull_websites_content(filename: str) -> None:
             messagebox.showinfo("Opps!", f"{e}")
```

Comment:
you can use the dedicated method for showing such errors 
besides, there's no reason to handle these errors independently, if the handling logic stays the same :P